src/app/components/ContactRow.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { addPlus, getInitial } from 'proton-shared/lib/helpers/string';
+
+import ItemCheckbox from './ItemCheckbox';
+import ContactGroupIcon from './ContactGroupIcon';
+
+const ContactRow = ({ style, contactID, hasPaidMail, contactGroupsMap, contact, onClick, onCheck, onStop }) => {
+    const { ID, Name, LabelIDs = [], emails = [], isChecked } = contact;
+
+    return (
+        <div
+            style={style}
+            key={ID}
+            onClick={onClick(ID)}
```

Comment:
To avoid to define a function like `const handleClick => (ID) => () => {}`, I try to adapt it here:
```suggestion
            onClick={() => onClick(ID)}
```